src/main/java/org/apache/datasketches/hll/TgtHllType.java

```diff
@@ -60,7 +60,7 @@ public enum TgtHllType {
    */
   HLL_6,
   /**
-   * An Hll Sketch with a bin size of 8 bits
```

Comment:
Really?  The HyperLogLog abbreviation appears in multiple cases both in code and comments:  hll, Hll, HLL.  And, yes, this could have been spelled all caps as HLL, but I don't think that it should cause any confusion.  

I appreciate your wanting to make a contribution, thank you, but it would be even more appreciated if you could find more substantive areas where improved comments would add clarity, and I am sure there are many :)

Reply:
By the way, I filed an issue at https://github.com/apache/datasketches-java/issues/688 when reading the code, I'll appreciate it if you can give a look :D

Reply:
As the background, here is how I easily find this inconsistency:

<img width="1179" height="2556" alt="IMG_6538" src="https://github.com/user-attachments/assets/94bea963-da7c-4077-9691-6cf22048e3c2" />

But I understand different projects have different preference. Thanks for your reply >~<